test/postman/dbp-test-collection.json

```diff
@@ -812,41 +894,59 @@
 							"raw": ""
 						},
 						"url": {
-							"raw": "http://{{url}}/activiti-app/api/enterprise/app-version",
-							"protocol": "http",
+							"raw": "{{apsUrl}}/api/enterprise/profile",
 							"host": [
-								"{{url}}"
+								"{{apsUrl}}"
 							],
 							"path": [
-								"activiti-app",
 								"api",
 								"enterprise",
-								"app-version"
+								"profile"
 							]
 						}
 					},
 					"response": []
 				}
+			],
+			"auth": {
+				"type": "noauth"
+			},
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "77beb5f0-bcc0-4cd1-abbe-06b51a16a847",
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"id": "ec22fea0-9e29-4139-91f2-9a14f94f3d8a",
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				}
 			]
 		},
 		{
-			"name": "auth gateway",
+			"name": "Auth Gateway routes",
 			"event": [
 				{
 					"listen": "test",
 					"script": {
+						"id": "db57447f-44f7-4e84-bbc2-7ed8ebf53c4b",
+						"type": "text/javascript",
 						"exec": [
-							"pm.globals.get(\"url\");",
```

Comment:
Wasnt this check done with token auth?

Reply:
This line was redundant, simply obtaining the value of global variable called "url" and subsequently doing nothing with it.